### PR TITLE
Added irc hangout template, made templates folder

### PR DESCRIPTION
**Pull request by RichardLitt**
Because this makes it easier for me.

### CODE REVIEW

File: templates/irc-hangout-announcement.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+========================== IPFS Hangouts: __topic__ =========================
+Topic: __topic__
```
Comment (diasdavid) [inline templates/irc-hangout-announcement.md]: 2x the topic?

Comment (RichardLitt) [inline templates/irc-hangout-announcement.md]: Nah, Sublime makes that easy.

Comment (diasdavid) [inline templates/irc-hangout-announcement.md]: :+1:

**Comment by RichardLitt** [comment]
Thanks!

**Pull request merged**